machine/delta/delta_ag5648/INSTALL

```diff
@@ -0,0 +1,86 @@
+================================================================================
```

Comment:
Question about the vendor name.  I just noticed this PR as well as #510 are using "delta", where the machine directory "dni" already exists.  I also see you use the "dni" in the code below.

What is is going to be going forward?  `delta` or `dni`?